### DISCUSSION TITLE
Welcome to CherryUSB Discussions!

### DISCUSSION THREAD

**Discussion by sakumisu** [cherry-embedded/CherryUSB#46] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by satar1980** [comment]
can you create example usb host mouse for f1c100s board ....

**Reply by sakumisu** [reply]
usb host demo supports all the devices.

**Reply by AshwinDeTaeye** [reply]
So also the rp2040?

**Reply by sakumisu** [reply]
> So also the rp2040?

Not, host porting is on going, device is avaiable.

**Reply by AshwinDeTaeye** [reply]
Great, can't wait for it. If you need help testing let me know.
Is there a way to sponsor the project?

**Reply by sakumisu** [reply]
> Great, can't wait for it. If you need help testing let me know. Is there a way to sponsor the project?

Thank you, just giving me star is well.
